Dockerfile

```diff
@@ -9,10 +9,10 @@ RUN apk --no-cache add \
     py3-pip \
     py3-wheel \
     python3 && \
-    pip install --upgrade pip==21.0.1
+    pip install --upgrade pip==23.3.2
 
 # Install a YAML Linter
-ARG yamllint_version=1.29.0
+ARG yamllint_version=1.33.0
```

Comment:
there is any breaking changes that migth affect the users?

Reply:
As far as I can tell, no. Relevant updates:

- one new rule (should be irrelevant unless users configure this, so not breaking)
- new rule options (also not breaking)
- fixed false-positives (meaning less errors, therefore not breaking)

Ref: https://github.com/adrienverge/yamllint/blob/master/CHANGELOG.rst

Reply:
thanks for checking